data-layout.md

```diff
@@ -0,0 +1,62 @@
+
+First, there are two relevant game directories:
+* `<install dir>`: The .exe file's install directory. You picked this when installing the game. Static game files and our added config files go here.
+* `<local data dir>`: `~/AppData/LocalLow/cygames/umamusume/`. The 'local data directory', where the game downloads new (and old) content.
+
+## Story/Lyrics etc.
+
+### Game
+
+The local data directory contains a file called `meta`, which is an sqlite database mapping story/winning live/etc. Unity files from a standardized filepath (presumably a path on the cygames servers) to a hash of its contents.
+
+These downloaded Unity files are stored in `<local data dir>/dat/` as files named with this hash, grouped into directories by the first two letters of each hash. For example, for some story data with hash `QOSUQ2JXFMHDA2LJJR2SDVBQ4LOGOKME`, it is found in the file `~/AppData/LocalLow/cygames/umamusume/dat/QO/QOSUQ2JXFMHDA2LJJR2SDVBQ4LOGOKME`.
+
+### Translations
+
+The various story/song/etc. files are organized in this repo's `translations` folder by a typically three-part 'story ID' parsed based on the structure of the server filepath mentioned in the above `meta` database. See [id-structure.md](id-structure.md) for a breakdown of the meaning of each part of the story ID.
+
+Json files storing the english translations for each of the game's data files - each of which is typically one scene - are stored in this repo at `translations/<text_type>/<group ID>/<sub-ID>/<idx>*.json`.
+
+E.g. `translations\story\04\1026\003 (譲れない答え).json`
+
+
+## master.mdb (skills, names, etc.)
+
+### Game
+
+Located at `~/AppData/LocalLow/cygames/umamusume/master/master.mdb`, this is a MariaDB database containing various non-dialogue data on the game, but including text data like menu screen text, item names, skill descriptions, etc.
+
+### Translations
+
+Stored in `translations/mdb`, these are organized as JSON files holding the various sub-types of data in `master.mdb`. Each contains JP to english text mappings.
+
+`src/mdb/index.json` maps each of these files to the master.mdb database table that they apply to (most being in a `text_data` table.
+
+To update `master.mdb`, it is simply searched for each of the JP text entries from these JSON files and replaced with the corresponding EN text.
+
+Note therefore that updates to existing translations can only be applied by restoring `master.mdb` so it contains the original JP text, and re-running `mdb import.bat`. This is easily achieved by deleting `~/AppData/LocalLow/cygames/umamusume/master/master.mdb` and re-launching the game; it will re-download it. Alternatively you can use `python mdb/import.py --backup` and `python mdb/import.py --restore` to backup/restore a copy of master.mdb for convenience.
+
+## UI (buttons, menu labels, etc.)
+
+### Game
+These are hardcoded in the exe or come from resource files in `<install dir>/umamusume_Data`, probably.
```

Comment:
Most are indeed in there, mainly `resources.assets`.
A few can be found in the local data dir I think. Nothing (outside of the icon...) is in the exe afaik.